### ISSUE TITLE
NVDA Magnifier - Moving the view should be smoother

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20060]
NVDA Magnifier updates the zoomed view upon tracking events (e.g. caret moved, mouse moved) or when panning commands are used.
When the view is updated, this update should be smoother, as in Windows Magnifier, rather than immediate, so that it be easier to track visually. This is probably more significant / important for people like myself who have a narrow central vision field.

### Steps to reproduce
* Open Notepad and type text
* Start NVDA magnifier and check that the caret is tracked at center.
* Navigate word by word with `control+right/leftArrows`

### Actual behavior

As soon as the caret is updated, the view is immediately updated to its new target position.

### Expected behavior
The view should be updated still quickly but in a more smooth way, so that the view update can be more easily tracked.

### NVDA logs, crash dumps and other attachments
Here are 2 screen capture videos at max magnification level, i.e. 10x to make things more obvious. One is done with Windows Magnifier and the other with NVDA Magnifier:

https://github.com/user-attachments/assets/8d3db0c1-ca73-4e4a-85a4-d9135d9cf85d

https://github.com/user-attachments/assets/44ec4795-69dd-41db-85ef-1e0d2becec92

If you open them, say in VLC, and lower the playing speed to 0.1, you can see easily that there is only one image update per caret move with NVDA Magnifier while with Windows Magnifier, there are intermediate images making the move tracking easier.

#### NVDA installed/portable/running from source
Installed
#### NVDA version
2026.1rc1 (=2026.1)
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6783
#### Name and version of other software in use when reproducing the issue
N/A
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors
No, magnifier is new.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested, unrelated.

**Comment by CyrilleB79** [comment]
Cc @Boumtchack.

**Comment by CyrilleB79** [comment]
Note: these tests were performed with 100% dpi scaling (setting Change the size of text, apps, and other items).

This issue may be considered a duplicate of #19987 but:
1. This one describe another consequence, i.e. caret tracking, with video comparison between Windows Magnifier and NVDA Magnifier.
2. In https://github.com/nvaccess/nvda/issues/19987#issuecomment-4384923139, @seanbudd mentions a higher dpi. This is not the case here.

**Comment by CyrilleB79** [comment]
@SaschaCowley you have tagget this as p5. To me this is a bit too low and does not reflect how important this request is.

I'd put at least a p4, matching the following point "UX inefficient (e.g. double speaking)" because a jerky movement of the magnifier is more penalizing for me than double speaking. The only point in p5 is "Other feature requests affecting a small subset of users" but this issue is not a feature request.

We could eveen put a p3, matching "Feature does not work as expected", since the mouse or text cursor should remain centered while it chatters at each move.

**Comment by CyrilleB79** [comment]
After thinking a bit more and to clarify, it becomes clear to me that smoother magnified view update should only apply to focus/caret, nav object and review cursor tracking. On the opposite, mouse tracking updates should not be smoothed at all so that the mouse cursor remain as much as possible steady in the center of the screen as it is the case for Windows Magnifier (when center option is selected of course).
